bugwarrior/services/redmine.py

```diff
@@ -82,10 +191,21 @@ def get_issue_url(self):
             self.origin['url'] + "/issues/" + six.text_type(self.record["id"])
         )
 
+    def get_converted_hours(self, estimated_hours):
+        tw = TaskWarriorShellout()
+        calc = tw._execute('calc', estimated_hours)
+        return (
+            calc[0].rstrip()
+        )
```

Comment:
The `estimated_hours` field is different from due date. Estimated hours comes from Redmine in the format `2`, `2.9`, `3.25`, etc. Here I am converting this integer value to a taskwarrior duration field.